### PR TITLE
fix(AdminSettings): server buttons visual interactivity

### PR DESCRIPTION
**Pull request by ShGKme**
### ☑️ Resolves

* A part of: https://github.com/nextcloud/spreed/issues/12800

### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
before | after

### 🚧 Tasks

- [x] Replace `tertiary-no-background` -> `tertiary`
  - Recording server - delete
  - Signaling server - delete
  - STUN server - delete (+ add a missing gap here)
  - TURN server - delete + test. 
- [x] Prevent infinite loading in TURN server testing
  - Don't switch to "testing" (loading) state if testing is not happening
    image
  - Disable the testing button when testing is not possible
    image
- [x] Align turn settings on center to avoid button stretch
  Before | After
  ---|---  
  image | image

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [x] 🖥️ Tested with Desktop client or should not be risky for it 
- [x] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/components/AdminSettings/TurnServer.vue [hunk 1]
```diff
@@ -63,7 +64,7 @@
 			</template>
 		</NcButton>
 		<NcButton v-show="!loading"
```
Comment (Antreesy) [inline src/components/AdminSettings/TurnServer.vue:66]: Add a `flex-shrink: 0` maybe, to make it square-ish?
image

**Review by Antreesy — APPROVED** [review]

File: src/components/AdminSettings/TurnServer.vue (hunk 1 above)
Comment (Antreesy) [reply src/components/AdminSettings/TurnServer.vue:67]: Nah, was caused by `align-items: stretch`

**Comment by ShGKme** [comment]
- Rebased onto `main`
- Fixed button stretch issue

File: src/components/AdminSettings/TurnServer.vue (hunk 1 above)
Comment (ShGKme) [reply src/components/AdminSettings/TurnServer.vue:67]: Fixed

**Review by Antreesy — APPROVED** [review]

**Pull request merged**